src/group_theory/subgroup.lean

```diff
@@ -1922,6 +1922,10 @@ two subgroups of an additive group are equal."]
 def subgroup_congr (h : H = K) : H ≃* K :=
 { map_mul' :=  λ _ _, rfl, ..equiv.set_congr $ congr_arg _ h }
 
+def subgroup_equiv_map {G'} [group G'] (e : G ≃* G') (H : subgroup G) :
+  H ≃* H.map e.to_monoid_hom :=
+by exact e.submonoid_equiv_map H.to_submonoid
```

Comment:
Is there an elaboration issue here? Does it work with `(e.submonoid_equiv_map H.to_submonoid : _)`?